.npmignore

```diff
@@ -3,27 +3,23 @@
 .editorconfig
 .eslintignore
 .eslintrc.json
-.gitignore
 .github
+.gitignore
 .npmignore
 .nycrc
 .project
 .settings
+.vagrant
+Dockerfile
+Gruntfile.js
+report.tap
 build/
 coverage/
 demos/
-Dockerfile
 examples/
-Gruntfile.js
 instrumented/
 products/
-report.tap
 reports/
-tests/component-tests/
-tests/configs/
-tests/framework-tests/
-tests/manual-tests/
-tests/node-tests/
-tests/test-core/testTests/
-tests/all-tests.html
-tests/testem.js
+tests/
+!tests/lib
+!tests/test-core
```

Comment:
I ran `npm pack` to see how things will be treated. However, it appears that the tests are all being included, not just tests/lib and tests/test-core

Reply:
Resolved in 2ba94f9.